### PR TITLE
add destructuring array into UseFetch type

### PR DESCRIPTION
**Pull request by dagda1**
I've added the destructured array typing to the `UseFetch` type.

### CODE REVIEW

File: src/useFetch.ts [hunk 1]
```diff
@@ -1,45 +1,9 @@
 import { useEffect, useState, useCallback, useRef, useContext, useMemo } from 'react'
 import URLContext from './URLContext'
-import { HTTPMethod } from "./types";
```
Comment (alex-cory) [inline src/useFetch.ts]: nit: remove `;` please :)

File: src/__tests__/useFetch.test.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ describe('use-http', () => {
 
   it('can be used without crashing', async () => {
     const div = document.createElement("div");
-    
+
     act(() => {
       ReactDOM.render(<TestApp />, div);
```
Comment (alex-cory) [inline src/__tests__/useFetch.test.tsx]: nit: remove `;` please :)

File: src/__tests__/useFetch.test.tsx [hunk 3]
```diff
@@ -79,7 +79,7 @@ describe('use-http', () => {
 
   it('can be used without crashing', async () => {
     const div = document.createElement("div");
```
Comment (alex-cory) [inline src/__tests__/useFetch.test.tsx]: nit: remove `;` please :)

File: src/__tests__/useFetch.test.tsx [hunk 4]
```diff
@@ -16,7 +16,7 @@ import {
   waitForElement
 } from '@testing-library/react'
 
-import {FetchMock} from "jest-fetch-mock";
+import { FetchMock } from "jest-fetch-mock";
 
 const fetch = global.fetch as FetchMock;
```
Comment (alex-cory) [inline src/__tests__/useFetch.test.tsx]: nit: remove `;` please :)

File: src/__tests__/useFetch.test.tsx [hunk 5]
```diff
@@ -16,7 +16,7 @@ import {
   waitForElement
 } from '@testing-library/react'
 
-import {FetchMock} from "jest-fetch-mock";
```
Comment (alex-cory) [inline src/__tests__/useFetch.test.tsx]: nit: remove `;` please :)

**Comment by dagda1** [comment]
hi alex, nits are totally fine.  it is your project.  you could use eslint with the fix switch to do the formatting for you but i don’t want to bombard you with multiple PRs.  feel free to disagree with anything.

i’ll fix the nits tomorrow

**Comment by alex-cory** [comment]
I'll just merge for now, we can clean up the rest later. :) #movefast

**Pull request merged**